core/src/main/resources/hudson/PluginManager/available.jelly

```diff
@@ -30,28 +30,28 @@ THE SOFTWARE.
   <l:layout title="${%Available Plugins} - ${%Plugin Manager}" permission="${app.SYSTEM_READ}">
     <j:set var="page" value="available"/>
 
-    <l:app-bar title="${%Plugin Manager}" />
-
-    <st:include page="sidepanel.jelly"/>
+    <l:side-panel>
+      <local:sidepanel xmlns:local="/hudson/PluginManager" page="available" />
```

Comment:
This is a little bit unrelated as the current behavior already shows this problems:

When I click on the side bar link I would assume that something happens immediately. However, it takes some seconds on my machine until the new view is rendered (due to the slow server query). I think it would improve the user experience if a small "loading..." text is shown in the table until the data has been received from the server. 

Additionally (some are existing problems): 
- it is not clear how many plugins have been loaded (and why only those) 
- how many plugins are available in total
- sorting is not applied after searching 
- several primary buttons

Reply:
Do you get delay on this page? I only get a delay on the installed plugins page, which is slower than it should be.

Reply:
On this page about 4 seconds and on the installed plugins 8 seconds. Typical UI guidelines recommend a progress indicator if an operation takes more than 1 second.

Reply:
(But I am running a lot on my machine right now, so this is not a good indicator)

Reply:
Right I certainly don't notice anything on the available page, installed is very noticeable though.

No harm to add a loading spinner though, but could be done separately.

Reply:
> This is a little bit unrelated as the current behavior already shows these problems:
> 
> When I click on the side bar link I would assume that something happens immediately. However, it takes some seconds on my machine until the new view is rendered (due to the slow server query). I think it would improve the user experience if a small "loading..." text is shown in the table until the data has been received from the server.
> 
> Additionally (some are existing problems):
> 
> * it is not clear how many plugins have been loaded (and why only those)
> * how many plugins are available in total
> * sorting is not applied after searching
> * several primary buttons

I've added these to my plugin manager to-do list - thanks, it's super appreciated!

Reply:
not sure if it's on your to-do or not, (possibly this https://github.com/jenkinsci/jenkins/pull/6783#discussion_r914917897)

but the select all / none at the bottom is very annoying.